specification/azsadmin/resource-manager/subscriptions/Microsoft.Subscriptions.Admin/preview/2015-11-01/Subscriptions.json

```diff
@@ -474,6 +504,240 @@
                 }
             }
         },
+        "IdentityHealthCheckReportDefinition": {
```

Comment:
Why not just `IdentityHealthCheckReport`?

Reply:
The 'IdentityHealthCheckReportDefinition' is consistent with the class name in source code. I followed the existing conventions like the 'CheckNameAvailabilityDefinition' in the Subscriptions.json.